htdocs/class/userutility.php

```diff
@@ -275,9 +275,12 @@ public static function getUnameFromIds($uid, $usereal = false, $linked = false)
             /** @var XoopsMySQLDatabase $xoopsDB */
             $xoopsDB = XoopsDatabaseFactory::getDatabaseConnection();
             $sql     = 'SELECT uid, uname, name FROM ' . $xoopsDB->prefix('users') . ' WHERE level > 0 AND uid IN(' . implode(',', array_unique($userid)) . ')';
-            if (!$result = $xoopsDB->query($sql)) {
+            $result = $xoopsDB->query($sql);
+            if (!$xoopsDB->isResultSet($result)) {
+//                \trigger_error("Query Failed! SQL: $sql- Error: " . $xoopsDB->error(), E_USER_ERROR);
```

Comment:
Why are you adding commented code?

Reply:
I'm just keeping the standard trigger, in case we decide to change to Exceptions or something else. 
Please note: this is NOT the final solution, it's just for us to review and come up with something that could become the standard solution.